data/design/agi_designers.json

```diff
@@ -0,0 +1,715 @@
+[
```

Comment:
I believe the standard format for corpora is to start with an object and then include the list as a named array. The objects generally also include "description" and "source" properties:

```javascript
{
    "description": "____________",
    "source": "____________",
    "designers": [ ]
}
```